app/Models/Configs.php

```diff
@@ -64,55 +66,49 @@ class Configs extends Model
 	/**
 	 * Sanity check.
 	 *
-	 * @param $value
+	 * @param string|null $value
 	 *
 	 * @return string
 	 */
-	public function sanity($value): string
+	public function sanity(?string $value): string
 	{
 		$message = '';
 		$val_range = [
 			self::BOOL => explode('|', self::BOOL),
 			self::TERNARY => explode('|', self::TERNARY),
 		];
 
+		$message_template_got = 'Error: Wrong property for ' . $this->key . ' in database, expected %s, got ' . ($value ?: 'NULL') . '.';
```

Comment:
Either the method, the parameter `$value` or this error message is confusing. The error message is worded as if `$value` has been read from the database. But the method `sanity` is called by method `set` with `$value` being equal to the value which Lychee wants to write into the database.